### PR TITLE
implement and test more /v1/tokens endpoints

### PR DESCRIPTION
**Pull request by iamigo**
- GET /v1/tokens/{key}
- DELETE /v1/tokens/{key}
- POST /v1/tokens/:id/revoke
- POST /v1/tokens/:id/restore

### CODE REVIEW

File: tests/api/v1/tokens/delete.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          `${tu.namePrefix}Voldemort`);
+        done();
+      }
+    });
+  });
+
+  it('not found', (done) => {
+    api.delete(`${path}/123-abc`)
+    .set('Authorization', '???')
+    .expect(constants.httpStatus.NOT_FOUND)
```
Comment (annyhe) [inline tests/api/v1/tokens/delete.js]: @iamigo do we expect a certain http code here?

File: tests/api/v1/tokens/revokeRestore.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      User.create({
+        profileId: profile.id,
+        name: `${tu.namePrefix}test@refocus.com`,
+        email: `${tu.namePrefix}test@refocus.com`,
+        password: 'user123password',
+      })
+    )
+    .then((user) => {
+      usr = user;
+      return Token.create({
```
Comment (annyhe) [inline tests/api/v1/tokens/revokeRestore.js]: @iamigo can do have a test where we expect the default value of token.isRevoked === 0?

**Comment by annyhe** [comment]
Also, how does someone post this through postman? I am getting an error
screen shot 2016-12-07 at 11 17 09 am
screen shot 2016-12-07 at 11 17 04 am

**Comment by annyhe** [comment]
any comments @shriramshankar ?

**Comment by iamigo** [comment]
@annyhe POST /v1/token - don't pass in a createdBy, just name; it figures out "createdBy" by decoding the authorization token. "sss" isn't a valid auth token--you can get an initial auth token using /v1/register. You can then use *that* token to create other tokens using POST /v1/token.

File: tests/api/v1/tokens/revokeRestore.js (hunk 2 above)
Comment (iamigo) [inline tests/api/v1/tokens/revokeRestore.js]: I'll add that expect(...) in my next PR for this story.

**Comment by annyhe** [comment]
@iamigo Docs may need changing: here's what this branch looks like on my local.
There's two token sections. Did you mean to combine them?
screen shot 2016-12-07 at 1 03 45 pm

**Comment by iamigo** [comment]
@annyhe docs are correct

**Pull request merged**